tests/bst_tests/test_hashtable.py

```diff
@@ -0,0 +1,69 @@
+from src.hashtable import HashTable
+
+
+def test_init_starts_up():
+    H = HashTable()
+    assert len(H._table) == 1024
+
+
+def test_table_builds():
+    H = HashTable(10)
+    assert H._table == [[], [], [], [], [], [], [], [], [], []]
+
+
+def test_hash_works_correctly():
+    H = HashTable(6)
+    assert H._hash("Hello World!") == 1085
+
+
+def test_set_works_correctly():
+    H = HashTable(6)
+    H.set("Hello World!", "Bubbles")
+    assert len(H._table[5]) == 1, H._table
+    assert len(H._table[0]) == 0, H._table
+    assert len(H._table[1]) == 0, H._table
+    assert len(H._table[2]) == 0, H._table
+    assert len(H._table[3]) == 0, H._table
+    assert len(H._table[4]) == 0, H._table
+
```

Comment:
Check that `H.set` also returns None when a key is called for that doesn't exist in the table